scripts/qa_crowbarsetup.sh

```diff
@@ -1254,6 +1254,9 @@ function onadmin_allocate()
         done
         wait
     fi
+    if [[ $cloud = qa1 ]] ; then
+        curl http://clouddata.cloud.suse.de/git/automation/scripts/qa1_nodes_reboot | bash
```

Comment:
There is no mkcloud_dir var in qa_crowbarsetup ... and possibly no git checkout either.